picker-starter/README.md

```diff
@@ -75,10 +81,11 @@ You should see this:
 
 ## Deploy
 
-Deploy the field plugin with the [CLI](https://www.npmjs.com/package/@storyblok/field-plugin-cli). Issue a [personal access token](https://app.storyblok.com/#/me/account?tab=token), rename `.env.local.example` to `.env.example`, open the file, set the value `STORYBLOK_PERSONAL_ACCESS_TOKEN`, and run
-
-From the **repository's root folder**, run the following command:
+Deploy your field plugin with the [CLI](https://www.npmjs.com/package/@storyblok/field-plugin-cli). Issue a [personal access token](https://app.storyblok.com/#/me/account?tab=token), rename `.env.local.example` to `.env.example`, open the file, set the value `STORYBLOK_PERSONAL_ACCESS_TOKEN`, and from the **plugin's root folder**, run the following command:
 
 ```shell
-yarn workspace picker-starter deploy
+yarn build
+
+# Deploy it using the CLI
+npx @storyblok/field-plugin-cli@latest deploy
```

Comment:
We already have `deploy` command in scripts of package.json. Why don't we put it here like `yarn deploy`?

Btw, I've check the package.json and

```
    "deploy": "npm run build && npx @storyblok/field-plugin-cli@latest deploy --name sb-picker-starter --scope partner-portal"
```

We should probably remove ` --name sb-picker-starter --scope partner-portal`. What do you think?

Reply:
I tried to keep the package.json file unchanged to avoid the need to modify also the [workflows/picker-starter.yml](https://github.com/storyblok/field-type-examples/blob/main/.github/workflows/picker-starter.yml), but since you mentioned it, I think it will be better to move these options from the `deploy script` to the workflow.

I did it [here](https://github.com/storyblok/field-type-examples/pull/90/commits/78eb2d6445c6072ddfd241e8b11178a6d98bc141)

Thanks for your suggestion, @eunjae-lee.

Reply:
@demetriusfeijoo ah that's why. yeah, now that parameters live on this root level of this repository, and the cloned project will no longer have `sb-picker-starter` stuff. Looks great! :D